src/cart/crt_init.c

```diff
@@ -116,6 +118,9 @@ static int data_init(int server, crt_init_options_t *opt)
 
 	D_DEBUG(DB_ALL, "initializing crt_gdata...\n");
 
+	if (crt_is_service())
+		setenv("UCX_IB_FORK_INIT", "n", 1);
```

Comment:
I think you would also want to check whether or not UCX is being used. It would just be cleaner in my opinion to keep OFI and UCX env settings separate. Also testing before if it has already been set and printing a message would also be helpful I think before you override it.

Reply:
I'll need to check if the provider has been established at this point in the code, if we need to not set this on servers running OFI (it shouldn't cause a problem to set it when OFI is used).  I pulled the UCX env vars out of the dump function for now, thinking that your suggestion of a subsequent env var clean up patch makes better sense.